pages/common/ftp.md

```diff
@@ -0,0 +1,23 @@
+# ftp
+
+> Internet File Transfer Program
+
+- Connect to a FTP server:
+
+`ftp {{url}}`
```

Comment:
perhaps using an example url (ending in example.com) would be more illustrative.